pkg/manifests/nginx.go

```diff
@@ -558,3 +571,19 @@ func newNginxIngressControllerHPA(conf *config.Config, ingressConfig *NginxIngre
 		},
 	}
 }
+
+func IsValidDefaultSSLCertSecret(certificate *v1alpha1.DefaultSSLCertificate) bool {
```

Comment:
we shouldn't need the validation in multiple places. We crd regex validation works. We shouldn't have to duplicate it